src/AutoUI/index.tsx

```diff
@@ -279,6 +307,33 @@ export const AutoUI = <T extends AutoUIBaseResource<T>>({
 			!!sdk?.tags,
 		[actions, sdk?.tags],
 	);
+
+	const internalOnChange = (
+		updatedFilters: JSONSchema[],
+		sortInfo: TableSortOptions<T> | null,
+		page: number,
+		itemsPerPage: number = 50,
+	) => {
+		if (!onChange) {
+			return;
+		}
+		const oData =
+			pagination?.serverSide && !!updatedFilters.length
```

Comment:
NIT: We can achieve the same with a single operator (or even w/o)
```suggestion
			pagination?.serverSide && updatedFilters.length > 0
```